plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouterArgs.sol

```diff
@@ -59,6 +59,7 @@ library PaymentInFlightExitRouterArgs {
         uint256 competingTxPos;
         bytes competingTxInclusionProof;
         bytes competingTxWitness;
+        bytes32 senderData;
```

Comment:
Do the integration docs and the `.md` files documenting the contracts reflect this change?

Reply:
Yep, will update them